apollo-router/src/plugins/telemetry/config_new/selectors.rs

```diff
@@ -3569,4 +3852,203 @@ mod test {
             Some("default".into())
         );
     }
+
```

Comment:
We need tests including both subgraph calls and connectors calls to make sure it works properly

Reply:
I added a test with both connectors and subgraph, let me know if it's sufficient.